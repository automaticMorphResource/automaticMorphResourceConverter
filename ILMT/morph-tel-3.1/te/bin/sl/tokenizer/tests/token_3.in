ఇంతటి అందాలను తనలో దాచుకున్న ఆస్ట్రేలియా ఖండంలో చూడదగ్గ విశేషాలను ఇప్పుడు తెలుసుకుందాం..! 
