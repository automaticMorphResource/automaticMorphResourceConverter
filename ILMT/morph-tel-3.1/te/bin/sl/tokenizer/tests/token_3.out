<Sentence id="1">
1	ఇంతటి	unk
2	అందాలను	unk
3	తనలో	unk
4	దాచుకున్న	unk
5	ఆస్ట్రేలియా	unk
6	ఖండంలో	unk
7	చూడదగ్గ	unk
8	విశేషాలను	unk
9	ఇప్పుడు	unk
10	తెలుసుకుందాం..	unk
11	!	unk
</Sentence>

